recipes/ipyopt/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ipyopt" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a105ef68227304c0bb81c7583cf9ea83d27bfc3584a7a2e531fdef991d49e03
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
```suggestion
  skip: true  # [win or py2k]
```

Reply:
Aha! I had `or py2` and it wasn't working. Will fix.